components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/PlacesBookmarksStorage.kt

```diff
@@ -111,7 +112,7 @@ open class PlacesBookmarksStorage(context: Context) : PlacesStorage(context), Bo
      * @param position The optional position to add the new node or null to append.
      * @return The guid of the newly inserted bookmark item.
      */
-    override suspend fun addItem(parentGuid: String, url: String, title: String, position: Int?): String {
+    override suspend fun addItem(parentGuid: String, url: String, title: String, position: UInt?): String {
```

Comment:
nit: Looks like `Int`->`UInt` is the only API-breaking change here, right? I guess it's technically more correct, but still would be nice to avoid this change if possible.